aep/general/0159/aep.md.j2

```diff
@@ -1,5 +1,69 @@
 # Reading across collections
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/13) for more
-information.
+Sometimes, it is useful for a user to be able to retrieve resources across
+multiple collections, or retrieve a single resource without needing to know
+what collection it is in.
+
+## Guidance
+
+APIs **may** support reading resources across multiple collections by allowing
+users to specify a `-` (the hyphen or dash character) as a wildcard character
```

Comment:
since we don't have a must on this resource path AEP:

```
Additionally, user-settable resource IDs should restrict letters to lower-case (^[a-z]([a-z0-9-]{0,61}[a-z0-9])?$).
```

We'll have resources which cannot adopt these patterns. Should we upgrade the above to a *must*?

It'd be great if we have guarantees around our APIs such that we can layer patterns on top of them painlessly, vs every one have some caveat of usability due to the fact that we weren't strong enough with our guarantees elsewhere.

Reply:
I don't know if a `**must**` is practical here.

Just as an example, Roblox would have to violate this in certain places (for legacy reasons).  Other organizations will likely run into the same thing.

But also - is the concern that a resource may be named `-`?  Otherwise I don't see the conflict.